CompuCell3D/core/CompuCell3D/steppables/PDESolvers/FluctuationCompensator.h

```diff
@@ -0,0 +1,161 @@
+/*************************************************************************
+*    CompuCell - A software framework for multimodel simulations of     *
+* biocomplexity problems Copyright (C) 2003 University of Notre Dame,   *
```

Comment:
maybe cr should be for 2020 IU. I think new software modules can get a "current year" cr in the place they were made